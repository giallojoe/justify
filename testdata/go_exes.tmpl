program:
	@bash -c 'for c in {{- range .GoExeCandidates }} "{{ . }}" {{- end }}; do echo "$c"; done'
