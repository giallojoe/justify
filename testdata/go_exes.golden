program:
	@bash -c 'for c in "bin1" "bin two" "bin3"; do echo "$c"; done'
